workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/ReaderWriterStore.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    internal class ReaderWriterStore
+    {
+        private readonly Dictionary<MonoBehaviour, Dictionary<uint, IReaderWriterInternal[]>> behaviourToReadersWriters
```

Comment:
behaviour to `?` to readerwriters

Reply:
Could call it something like behaviourToReadersWritersForComponents or something similar, but I feel that would be too long to be readable... I think most of these were changed when @jared-improbable suggested the original names I put in the BehaviourManager PR were too verbose, and especially here I think making this fully expressive would just return to that problem

Reply:
Understood, not a big issue, resolving